docs/install/linux.rst

```diff
@@ -25,23 +25,24 @@ You need to be familiar with command-line interfaces to get through this comfort
 
 Short version
 =============
-.. parsed-literal::
+::
 
-  curl -sL https\:\/\/deb.nodesource.com/setup_12.x | sudo -E bash -    # add Node.js source repository
-  sudo apt install python3-dev python3-venv postgresql libpq-dev nodejs gcc g++ make
-  git clone https\:\/\/github.com/TabbycatDebate/tabbycat.git
+  curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -    # add Node.js source repository
+  sudo apt install python3-dev pipenv postgresql libpq-dev nodejs gcc g++ make
+  git clone https://github.com/TabbycatDebate/tabbycat.git
   cd tabbycat
   git checkout master
-  sudo -u postgres createuser myusername --createdb --pwprompt    # skip if not first time
+  sudo -u postgres createuser myusername --createdb --pwprompt       # skip if not first time
   createdb mydatabasename
 
 Then create **settings/local.py** as described :ref:`below <local-settings-linux>`, then::
 
-  python3 -m venv venv
-  source venv/bin/activate
-  pip install --upgrade pip
-  pip install -r ./config/requirements_core.txt
+  pipenv install
   npm install
```

Comment:
Should be `npm ci` to install from lock rather than install fresh.

Reply:
```suggestion
  npm ci
```

Reply:
Hmm, if we do this then I guess `pipenv install` should also be `pipenv install --deploy` everywhere it occurs in the instructions, and also in the Dockerfile. That is, all installation instructions should install from lock.

I think I'm on board with this, all things considered. Installation instructions are at least ostensibly aimed at users, not developers, and advanced users who want to update these can do so themselves (and at their own risk). @philipbelesky, thoughts?

Reply:
Sounds good. I don't think we've been bitten by any dependency issues from not using the lock files, but makes sense to be consistent/deterministic in the installs.